templates/app/app.hbs

```diff
@@ -20,7 +20,7 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-let allowedOrigins = [/\.forestadmin\.com$/{{#if forestUrl }}, /localhost:\d{4}$/{{/if}}];
```

Comment:
Silly question.
Couldn't we keep the original `allowedOrigins` for all routes except sessions ones.
And do something like this for sessions routes:
```javascript
const originsForSessions = allowedOrigins.concat('null');
app.use('/forest/sessions', cors({
  origin: originsForSessions,
  allowedHeaders: ['Authorization', 'X-Requested-With', 'Content-Type']
  maxAge: 86400, // NOTICE: 1 day
  credentials: true,
}))

app.use(cors({ /* ... */ }));
```

Reply:
Yes we can.

Reply:
The new routes for oidc are under `/forest/authentication`